helm/websub-consolidator/values.yaml

```diff
@@ -12,18 +12,23 @@
 ##
```

Comment:
repository: mosipid/consolidator-websub-service
  tag: 1.3.0-beta.1 
to
  repository: mosipqa/consolidator-websub-service
  tag: 1.3.x

javaOpts: "-Xms750M -Xmx750M" 
to
javaOpts: "-Xms1200M -Xmx1200M"